gettsim/tax_transfer.py

```diff
@@ -510,9 +542,96 @@ def _apply_squeeze_function(group, tax_func, level, func_args, func_kwargs):
         return tax_func(group, *func_args, **func_kwargs)
 
 
-def calculate_tax_and_transfers(dataset, policy_year):
-    tax_data = get_policies_for_date(year=policy_year)
-    tax_data_pensions = pd.read_excel(ROOT_DIR / "data" / "pensions.xlsx").set_index(
-        "var"
+def calculate_tax_and_transfers(
+    dataset,
+    year,
+    day=1,
+    month=1,
+    arbeitsl_geld_2_params=None,
```

Comment:
I would like to avoid these defaults at this point. This is our user-facing function for the moment. Let us not mess with it and leave the interface in place unless there are reasons I missed.

Reply:
I included day and month, because the code works for every date in the year, so I thought why should we just allow for the year and not allow for every date. The other defaults are if anybody wants to define own policy data. I can delete if you like....

Reply:
Yes, please do. We do not have a way for selecting things other than Jan 1 for the moment. So we should not give the user the impression that such a thing might work.

Same for the rest. Setting up an interface for reforms will be complex, see for example here for OpenFisca's approach: https://openfisca.org/doc/openfisca-python-api/reforms.html. We should not just add something because it is easy. 

For now, our scope is to provide an accurate depiction of the current (and past) Taxes and Transfers system. Next, an accurate fast depiction. Then allow users to modify.

Reply:
I deleted it. But @mjbloemer wanted to update the dates in the yaml files and policy for dates would actually select on the date. But you are right: We would pretend to be daily accurate but actually we are not!

Reply:
Yes, of course we will eventually do this. But we do not at present.